README.md

```diff
@@ -199,15 +199,49 @@ This is an acceptable way to contribute images, as long as the size is kept smal
 
 > Note: Do NOT put videos inside this directory! Upload the video into your preferred online storage system such as Google Drive, DropBox, or YouTube, and share its link with us to review and put it up online.
 
+Put images in separate folders within the main "images" directory, one for each top level section of the documentation. 
```

Comment:
We should also add links (and a quick description) to "Components", "Workflow Structure" into the Overview page for PythonML, otherwise it looks empty now: https://docs.exabyte.io/software-directory/machine-learning/python-ml/overview/

Reply:
Added these descriptions